voila/configuration.py

```diff
@@ -85,3 +85,11 @@ class VoilaConfiguration(traitlets.config.Configurable):
     show_tracebacks = Bool(False, config=True, help=(
         'Whether to send tracebacks to clients on exceptions.'
     ))
+
+    multi_kernel_manager_class = Type(
+        default_value='jupyter_server.services.kernels.kernelmanager.AsyncMappingKernelManager',
+        klass='jupyter_client.multikernelmanager.MultiKernelManager',
+        help="""The kernel manager class. This is useful to specify a different kernel manager,
+        for example a kernel manager with support for pooling.
+        """
+    ).tag(config=True)
```

Comment:
`config=True` should be a kwarg, not a tag (the current behavior will work but is deprecated).

Reply:
Hmm yes, I see we do that a lot, lets fix that in a different PR ok?